.gitignore

```diff
@@ -28,6 +28,13 @@ var/
 .ipynb_checkpoints
 .virtual_documents
 
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+
 # PyInstaller
```

Comment:
Other options that come to my mind are 

* `hatch.toml`
* `pdm.lock`
* `Pipfile.lock`
* `Pipfile`
* `poetry`.lock`

I'm not completely sure if people should be building ArviZ following an approach different from the recommended one, but people can do whatever they want so it's probably good to have these files/directories listed.

Reply:
Yes, I believe it's important to use the recommended approaches. However, when installing dependencies, some may prefer to use a virtual environment. While it’s important to track the core dependency and configuration files like Pipfile, Pipfile.lock, poetry.lock, pdm.lock, and hatch.toml in version control, it’s equally important to ignore files and directories that are specific to individual developers' environments, such as virtual environments. These files are not relevant to other people's builds and can clutter the repository.